src/esn.inbox.libs/app/services/mailboxes/mailboxes-service.spec.js

```diff
@@ -1491,9 +1491,12 @@ describe('The inboxMailboxesService factory', function() {
   describe('The updateSharedMailboxCache function', function() {
 
     it('should do anything on inboxMailboxesCache', function(done) {
-      inboxMailboxesCache = [];
+      inboxMailboxesCache = { state: '1', list: [] };
```

Comment:
Should set the cache with
```suggestion
      inboxMailboxesCache.state = '1';
      inboxMailboxesCache.list = [];
```